WordPress/Classes/ViewRelated/Plugins/Views/CollectionViewContainerRow.swift

```diff
@@ -73,19 +73,20 @@ struct CollectionViewContainerRow<CollectionViewCellType: UICollectionViewCell,
 
         cell.noResultsView = noResultsView
 
-        if nil != Bundle.main.path(forResource: String(describing: CollectionViewCellType.self), ofType: "nib") {
-            cell.collectionView.register(UINib(nibName: String(describing: CollectionViewCellType.self), bundle: nil),
-                                         forCellWithReuseIdentifier: CollectionViewHandler.CellReuseIdentifier)
+        let bundle = Bundle(for: BundleToken.self)
```

Comment:
Could this be `Bundle.keystone`?

```suggestion
        let bundle = Bundle.keystone
```

Reply:
I'll update it later. It was from the first batch of the changes I made before adding `Bundle.keystone`.